mypy_django_plugin/main.py

```diff
@@ -52,25 +52,28 @@ def add_new_manager_base(ctx: ClassDefContext) -> None:
 
 
 def extract_django_settings_module(config_file_path: Optional[str]) -> str:
-    errors = Errors()
+    usage = "\n".join(["(config)",
+                       "[mypy.plugins.django_stubs]",
+                       "\t(required) django_settings_module: str"])
+
+    def raise_error(handler: ErrorHandler, error_type: int) -> NoReturn:
+        messages = {1: "'django_settings_module' is not set: no mypy config file specified",
+                    2: "'django_settings_module' is not set: no section [mypy.plugins.django-stubs]",
+                    3: "'django_settings_module' is not set: setting is not provided"}
+        handler.error(messages[error_type])
+
+    error = partial(raise_error, ErrorHandler(prog="(django-stubs) mypy", usage=usage))
     if config_file_path is None:
-        errors.report(0, None, "'django_settings_module' is not set: no mypy config file specified")
-        errors.raise_error()
+        error(1)
 
     parser = configparser.ConfigParser()
-    parser.read(config_file_path)  # type: ignore
-
-    if not parser.has_section('mypy.plugins.django-stubs'):
-        errors.report(0, None, "'django_settings_module' is not set: no section [mypy.plugins.django-stubs]",
-                      file=config_file_path)
-        errors.raise_error()
-    if not parser.has_option('mypy.plugins.django-stubs', 'django_settings_module'):
-        errors.report(0, None, "'django_settings_module' is not set: setting is not provided",
-                      file=config_file_path)
-        errors.raise_error()
-
-    django_settings_module = parser.get('mypy.plugins.django-stubs', 'django_settings_module').strip('\'"')
-    return django_settings_module
+    parser.read(cast(str, config_file_path))
+
+    section = 'mypy.plugins.django-stubs'
```

Comment:
Can we reuse this const from somewhere else? Looks like we should already have it somewhere.

Reply:
Doesn't seems like it - unless not anywhere in the source code:
![image](https://user-images.githubusercontent.com/16212750/87087173-88eefc80-c22a-11ea-89c0-d77a4376d816.png)